data/categories.txt

```diff
@@ -2162,7 +2162,7 @@ fr:Arrêt de tramway|3tramway|tram|arrêt
 de:4Straßenbahnhaltestelle|Tramhaltestelle|Haltestelle|Tram|ÖPNV
 hu:4Villamosmegálló|3villamos|megálló
 id:3Perhentian trem
-it:Fermata|tram
+it:Fermata del tram|tram
```

Comment:
NOTE: I'm not 100% sure that I'm respecting the logic, but it's not just "Fermata" but "Fermata del tram" if related to a tram.